assume-role

```diff
@@ -250,11 +330,15 @@ assume-role(){
     AWS_ACCESS_KEY_ID=$(echo "$ROLE_SESSION" | jq -r .Credentials.AccessKeyId)
     AWS_SECRET_ACCESS_KEY=$(echo "$ROLE_SESSION" | jq -r .Credentials.SecretAccessKey)
     AWS_SESSION_TOKEN=$(echo "$ROLE_SESSION" | jq -r .Credentials.SessionToken)
-    AWS_SECURITY_TOKEN=$AWS_SESSION_TOKEN
-    AWS_ACCOUNT_ID="$account_id"
-    AWS_ACCOUNT_NAME="$account_name"
-    AWS_ACCOUNT_ROLE="$role"
-    GEO_ENV="$account_name" # For GeoEngineer https://github.com/coinbase/geoengineer
+    export AWS_ACCESS_KEY_ID
```

Comment:
do we need these exported now?

Reply:
What do you mean by that? I am just following `shellcheck`'s conventions. This is the `export-envars` function, so this is done after you successfully assume-role, so I would say that yes, this is the right place to export those envars.

Reply:
Unless you want to export it first, ie
```
export AWS_ACCESS_KEY_ID # in setup
# some code here, ie assume-role flow
AWS_ACCESS_KEY_ID=...
```